usecase/parser/block.go

```diff
@@ -12,6 +13,7 @@ import (
 
 func ParseBlockToCommands(
 	parserManager *utils.CosmosParserManager,
+	logger applogger.Logger,
```

Comment:
Do you think it make sense to use the `Logger` from parserManager rather a new one?

Reply:
Agree, I update it.